include/json/writer.h

```diff
@@ -360,8 +360,8 @@ JSON_API OStream& operator<<(OStream&, const Value& root);
 
 #pragma pack(pop)
 
-#if defined(JSONCPP_DISABLE_DLL_INTERFACE_WARNING)
+#if defined(JSONCPP_DISABLE_DLL_INTERFACE_WARNING) && defined(_MSC_VER)
 #pragma warning(pop)
-#endif // if defined(JSONCPP_DISABLE_DLL_INTERFACE_WARNING)
+#endif // if defined(JSONCPP_DISABLE_DLL_INTERFACE_WARNING) && defined(_MSC_VER)
```

Comment:
Let's just delete the comments on these `#endif` statements if the block is small and leafy like this. It's lame to keep them up to date as the `#if` condition evolves.

Reply:
I just looked at json/src/config.h and we already do what I'm requesting.
The JSONCPP_DISABLE_DLL_INTERFACE_WARNING is only defined if _MSC_VER is defined.
This is by design. I don't think your PR will improve anything. Do you have a motivating case like a build failure you can provide?